ansible/group_vars/all.yml

```diff
@@ -65,3 +65,8 @@ shiftstack_local_storage_node_selector: beta.kubernetes.io/instance-type=nvme-8
 shiftstack_local_storage_device: /dev/nvme0n1
 shiftstack_thin_volume_count: 6
 shiftstack_thin_volume_size: 140g
+  
```

Comment:
mega-nitpick here but could we get the rest of the "divider" `################################################################################`